web/client/translations/data.it-IT

```diff
@@ -1305,6 +1305,23 @@
                 "pie": "Mostra legenda",
                 "bar": "Mostra legenda",
                 "gauge": "Mostra etichette"
+            },
```

Comment:
use google translate for the translations for it, es, fr, de
the other languages you can leave english